themes-default/slim/test/specs/app-link.spec.js

```diff
@@ -0,0 +1,87 @@
+import test from 'ava';
+import Puex from 'puex';
+import { mount } from 'vue-test-utils';
+
+// Needs to be required otherwise nyc won't see it
+const AppLink = require('../../views/vue-components/app-link.vue');
+
+test.beforeEach(t => {
+    t.context.base = 'http://localhost:8081/';
```

Comment:
Please make the base configurable using environment variables or command line, falling back to `localhost:8081`.
This is a pain I have with the current `dredd` setup.